metron-interface/metron-config/README.md

```diff
@@ -0,0 +1,67 @@
+# Metron Management UI
+
+This module provides a user interface for management functions in Metron.
+
+## Prerequisites
+
+* A network accessible Metron REST application
+* npm 3.8.9+ (npm can be installed on quick dev with `yum install npm && npm install npm@3.8.9 -g`)
+
+## Installation
+1. Build Metron:
+    ```
+    mvn clean package -DskipTests
+    ```
+  
+1. Copy `incubator-metron/metron-interface/metron-config/target/metron-config-METRON_VERSION-archive.tar.gz` to the desired host.
+
+1. Untar the archive in the target directory.  The directory structure will look like:
```

Comment:
My interpretation of $METRON_HOME as used in our docs is the deployment path (/usr/metron/0.3.1 for example).  This path is the relative path to the metron-config jar on the build machine.  What convention should I use to make this clearer?